include/lightweight_filtering/Update.hpp

```diff
@@ -238,7 +243,14 @@ class Update: public ModelBase<Update<Innovation,FilterState,Meas,Noise,OutlierD
     // Outlier detection // TODO: adapt for special linearization point
     outlierDetection_.doOutlierDetection(innVector_,Py_,Hlin_);
     Pyinv_.setIdentity();
-    Py_.llt().solveInPlace(Pyinv_);
+
+    Eigen::LLT<Eigen::MatrixXd> llt_py(Py_);
+    if (llt_py.info() != Eigen::Success) {
+      LOG(WARNING)  << "LLT failed.";
```

Comment:
rm double space after (WARNING)

Reply:
btw, would it make sense to print the condition number in this warning to have an idea how degenerate the setting is?

Reply:
VLOG1